tests/utils/set_params.cpp

```diff
@@ -782,13 +787,16 @@ void setMultigridEigParam(QudaEigParam &mg_eig_param, int level)
   }
 
   mg_eig_param.ortho_block_size = mg_eig_ortho_block_size[level];
+  mg_eig_param.compute_evals_batch_size = mg_eig_evals_batch_size[level];
   mg_eig_param.block_size
     = (mg_eig_param.eig_type == QUDA_EIG_TR_LANCZOS || mg_eig_param.eig_type == QUDA_EIG_IR_ARNOLDI) ?
     1 :
     mg_eig_block_size[level];
   mg_eig_param.n_ev = mg_eig_n_ev[level];
   mg_eig_param.n_kr = mg_eig_n_kr[level];
   mg_eig_param.n_conv = nvec[level];
+  mg_eig_param.compute_evals_batch_size
```

Comment:
This variable is being set twice

Reply:
[7903288](https://github.com/lattice/quda/pull/1489/commits/7903288629f0fcc474989fec5a1393ecc17a4b42)